internal/stack/_static/docker-compose-stack.yml.tmpl

```diff
@@ -170,11 +170,12 @@ services:
       interval: 60s
       timeout: 50s
       retries: 5
-    command: bash -c 'if [[ ! $(bin/logstash-plugin list) == *"logstash-filter-elastic_integration"* ]]; then echo "Missing plugin logstash-filter-elastic_integration, installing now" && bin/logstash-plugin install logstash-filter-elastic_integration; fi && bin/logstash -f /usr/share/logstash/pipeline/logstash.conf'
+    command: bash -c 'chmod +x /usr/share/logstash/startup.sh && /usr/share/logstash/startup.sh'
```

Comment:
Let me ping @dliappis here. Need your feedback for the case - privilege required `chmod` fails with below error on the BK agent (perfectly fine on my local machine). The issue here `/usr/share/logstash/startup.sh` is mounted with different user and another non-privileged user is trying to `chmod`. Some ideas (add init service with root, mount & chmod OR change ownership to logstash user) I have but I don't know if they work/feasible (or which one will be a right way). It is a `WINDOWS_AGENT_IMAGE="family/ci-windows-2022"`
```
logstash-1  | chmod: changing permissions of '/usr/share/logstash/startup.sh': Operation not permitted
```

Reply:
@mashhurs I am a bit confused, you mentioned this is failing on a ... Windows agent? (`It is a WINDOWS_AGENT_IMAGE="family/ci-windows-2022"`)

If yes, I'd need a little more context about what's going on here. Are we binding mounting a directory/file from the agent os and then attempting to `chmod +x` the file inside a container?

Reply:
Are we binding mounting a directory/file from the agent os and then attempting to `chmod +x` the file inside a container?
- Correct!
As my understanding, a file permission _usually_ gets inherited if the file is mounted. For this case, I also didn't know that we have a middleman, the [`go-resource` seems deciding its permission](https://github.com/elastic/elastic-package/pull/1668#discussion_r1489470243). I was wondering if permission was updated or ownership got changed by BK agents since I had no idea why the permission was changed.
Anyway thank you and sorry for bother you here.